Assets/HoloToolkit/Input/Scripts/HandGuidance.cs

```diff
@@ -14,20 +14,21 @@ namespace HoloToolkit.Unity.InputModule
     /// </summary>
     public class HandGuidance : Singleton<HandGuidance>
     {
-#if UNITY_WSA
         [Tooltip("The Cursor object the HandGuidanceIndicator will be positioned around.")]
         public GameObject Cursor;
```

Comment:
You should also move the compiler directive for the `using UnityEngine;` at the top so that `GameObject` and `Tooltip` are usable.